prescient/engine/data_extractors.py

```diff
@@ -100,6 +100,11 @@ def get_power_generated_T0(self, sced: OperationsModel, g: G) -> float:
         """Get the amount of power generated by a specific generator in the timestep before this model."""
         pass
 
+    @abstractmethod
+    def get_bus_demand(self, sced: OperationsModel, bus: B) -> float:
+        """Get the demand at a specific bus."""
+        pass    
```

Comment:
At first I was wondering where the implementation of this method was, but it already exists in the Egret ScedDataExtractor.

Reply:
I was wondering the same thing when I started thinking about this change :)